use-cases-and-requirements.bs

```diff
@@ -0,0 +1,219 @@
+<pre class='metadata'>
+Title: Solid Use Cases and Requirements
+Shortname: your-spec
+Level: 1
+Status: w3c/ED
+Group: solid-cg
+URL: http://example.com/url-this-spec-will-live-at
+Editor: elf Pavlik
+Abstract: Use Cases and Requirements for Solid ecosystem.
+</pre>
+
+Introduction {#intro}
+=====================
+
+Issue: Needs explaination how proposals should clearly reference requirements which they satisfy. As well as requirements they do not satisfy due to some limitations.
+
+Requirements {#requirements}
+============================
+
+## The system is not abusable
+
+<div class="assertion" heading="Use cases">
+* [[#evil-fitness-tracker]]
+</div>
+
+## An app can request access to a specific resource
+
+<div class="assertion" heading="Use cases">
+* [[#simple-game]]
+* [[#rdf-editor]]
+* [[#pod-administration]]
+</div>
+
+## An app can request access to a specific type of data without knowing the structure of resources on a Pod
+
+<div class="assertion" heading="Use cases">
+* [[#chat-with-friends]]
+* [[#discussion-boards]]
+* [[#photos-viewer]]
+* [[#photos-processing]]
+</div>
+
+## Access requests can be sent when the resource owner is not present to be approved once the user is present
+
+## Apps can request the ability to write a specific type of data and will be told where it should write it
+
+<div class="assertion" heading="Use cases">
+* [[#photos-processing]]
+</div>
+
+## It should be possible for an agent to block/allow certain apps from accessing a specific resource as that agent
+
+## It should be possible for an agent with Control access to block/allow certain apps from accessing a specific resource as any agent
+
+<div class="assertion" heading="Use cases">
+* [[#chat-with-doctor]]
+</div>
+
+## It should be easy to allow others accessing your resources to use apps you're okay without requiring your explicit consent.
+
+## Access to specific types of data should extend to new resources that contain that data
+
+## Access to specific types of data should not expose other data that was not requested
+
+## Data should have different levels of requirements for user's conciousness in consent
+
+## The authorization data is easily cachable
+
+## Information about Clients (apps) to which users have granted access and what specific access they have delegated should not be made available to Resource Servers that do not enforce relevant access restrictions on those Clients.
+
+
+Use Cases {#use-cases}
+======================
+
```

Comment:
It feels like the use cases should be organized more based on the unique pattern that they're demonstrating, so they can be found and navigated more intuitively.